wip-mariocao-median.md

```diff
@@ -0,0 +1,98 @@
+<pre>
+  WIP: WIP-0015
```

Comment:
Please wait for a WIP number to be assigned

Reply:
The WIP editor is not happy.

Reply:
Ups...that was a copy paste as it is already in use! :+1: